management/guardian_test.go

```diff
@@ -12,267 +13,314 @@ import (
 func TestGuardian(t *testing.T) {
 	t.Run("MultiFactor", func(t *testing.T) {
 		t.Run("List", func(t *testing.T) {
+			const expectedGuardianFactors = 8
```

Comment:
Where does this number come from? If it's from the API, shouldn't we assert that it is > 0 so the tests don't break when new factors are added, or some get dropped?

Reply:
It's just from the API:) good point! I updated this in [52df276](https://github.com/auth0/go-auth0/pull/53/commits/52df276c920f66ce5fa72a33cd9dc367fe3929bf)